LiskHQ/lisk-explorer
# Update the UI to align with new Lisk design - Closes #442

### What was the problem?
Detailed description in #442

### How to test it?
Make sure update aligns with the UI design of Lisk Hub and Lisk.io
This PR doesn't include any changes regarding responsiveness in smaller screens.

### Review checklist
- The PR solves #442 
- All new code follows best practices
